src/test/java/org/threeten/extra/chrono/TestInternationalFixedChronology.java

```diff
@@ -0,0 +1,1097 @@
+        assertEquals(InternationalFixedChronology.INSTANCE.dateYearDay(2004, 169), InternationalFixedDate.leapDay(2004));
+        assertEquals(InternationalFixedChronology.INSTANCE.dateYearDay(2004, 366), InternationalFixedDate.yearDay(2004));
+    }
+
+    private void checkYearDateForYear(final int year, final int daysInYear) {
+        InternationalFixedDate date = null;
+        int month = 0;
+        int dayOfMonth = 0;
+        int y = 0;
+
+        for (int i = 1; i <= daysInYear; i++) {
+            date = InternationalFixedChronology.INSTANCE.dateYearDay(year, i);
+            assertEquals(date.getDayOfYear(), i);
+
+            if (date.isYearDay() || date.isLeapDay()) {
+                // year day / leap day are not part of any months!
+                dayOfMonth = 0;
+                month = date.isLeapDay() ? -1 : 0;
+            }
+            else {
+                y = (i > 169 && date.isLeapYear()) ? i - 1 : i;
+                dayOfMonth = ((y - 1) % 28) + 1;
+                month = ((y - 1) / 28) + 1;
+            }
+
+            assertEquals(month, date.getMonth());
+            assertEquals(dayOfMonth, date.getDayOfMonth());
+        }
+    }
+
+    //-----------------------------------------------------------------------
+    // InternationalFixedDate.isLeapYear
+    //-----------------------------------------------------------------------
+    @Test
+    public void test_isLeapYear_loop() {
+        Predicate<Integer> isLeapYear = year -> {
+            return ((year & 3) == 0) &&((year % 100) != 0 ||(year % 400) == 0);
+        };
+
+        for(int year = 1; year < 1001; year++) {
+            InternationalFixedDate base = InternationalFixedDate.of(year, 1, 1);
+            assertEquals(base.isLeapYear(), isLeapYear.test(year), "Year " + year + " is failing");
+            assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(year), isLeapYear.test(year), "Year " + year + " is failing leap-year test");
+        }
+    }
+
+    @Test
+    public void test_isLeapYear_specific() {
+        assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(400), true);
+        assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(100), false);
+        assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(4), true);
+        assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(3), false);
+        assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(2), false);
+        assertEquals(InternationalFixedChronology.INSTANCE.isLeapYear(1), false);
+    }
+
+    //-----------------------------------------------------------------------
+    // InternationalFixedDate.era
+    // InternationalFixedDate.dateYearDay
+    // InternationalFixedDate.prolepticYear
+    //-----------------------------------------------------------------------
+    @DataProvider(name = "lengthOfMonth")
+    Object[][] data_lengthOfMonth() {
```

Comment:
The `lengthOfMonth` data used to contain this: 

    368  +    { 1901, 13, 28 }, 
    369  +    { 1902, 13, 28 }, 
    370  +    { 1903, 13, 28 }, 
    371  +    { 1904, 13, 28 }, 
    372  +    { 1905, 13, 28 }, 
    373  +    { 1906, 13, 28 }, 
    374  +    { 2000, 13, 28 }, 
    375  +    { 2100, 13, 28 },    

... but you've since removed it.